### PR TITLE
Created branch lesson-2. Installed Node.js and Prettier.

### PR DESCRIPTION
**Pull request by KateRadziukevich**
node_modules/ add in .gitignore
Created directory about and created a file index.html
In the file index.html wrote a text about why you chose the direction of Front-end development and used various tags

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .idea
+node_modules
```
Comment (5tronciy) [inline .gitignore]: node_modules and node_modules/ are almost the same.

File: index.html [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+  <h1>Моя первая страница</h1>
+  <p>
+    <a href="./about">Об авторе</a>
```
Comment (5tronciy) [inline index.html]: wrong path

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+  </head>
+  <body>
+    <h1>Почему Front-end разработка?</h1>
+    <p>
+      <blockquote cite="https://ru.citaty.net/tsitaty/463117-linus-torvalds-zdes-ia-khochu-rasskazat-o-svoikh-zolotykh-pravilakh/">
+      <em>„Здесь я хочу рассказать о своих золотых правилах. Первое: обращайся с другими так, как ты хочешь, чтобы они обращались с тобой. Следуя этому правилу, в любой ситуации будешь знать, что делать. Второе: гордись тем, что делаешь. Третье: делай все с удовольствием.“ —Линус Торвальдс.</em>
+    </blockquote>
```
Comment (5tronciy) [inline about/index.html]: bad style: open and close tags should be on the same level.

File: about/index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+  </head>
+  <body>
+    <h1>Почему Front-end разработка?</h1>
+    <p>
+      <blockquote cite="https://ru.citaty.net/tsitaty/463117-linus-torvalds-zdes-ia-khochu-rasskazat-o-svoikh-zolotykh-pravilakh/">
+      <em>„Здесь я хочу рассказать о своих золотых правилах. Первое: обращайся с другими так, как ты хочешь, чтобы они обращались с тобой. Следуя этому правилу, в любой ситуации будешь знать, что делать. Второе: гордись тем, что делаешь. Третье: делай все с удовольствием.“ —Линус Торвальдс.</em>
```
Comment (5tronciy) [inline about/index.html]: Too long string, don't forget to use prettier.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Added some comments in rows

**Comment by 5tronciy** [comment]
You should make changes before requesting new review.

**Comment by 5tronciy** [comment]
Looks better. But why did you fix not all remarks?

File: .gitignore (hunk 1 above)
Comment (KateRadziukevich) [reply .gitignore]: I do not know what to do with the remark

Comment (5tronciy) [reply .gitignore]: Read the task, pls.

File: index.html (hunk 2 above)
Comment (5tronciy) [reply index.html]: Does it work in your browser?

Comment (KateRadziukevich) [reply index.html]: Yes

Comment (5tronciy) [reply index.html]: I checked it in Chrome and Safari - it is not what you expected. Websites are for people, not only for you.

**Pull request merged**